boot/bootutil/src/bootutil_swap_priv.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2019 JUUL Labs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "mcuboot_config/mcuboot_config.h"
+
+#if defined(MCUBOOT_SWAP_USING_SCRATCH) || defined(MCUBOOT_SWAP_USING_MOVE)
+
+int boot_erase_trailer_sectors(const struct boot_loader_state *state,
+                               const struct flash_area *fap);
+int boot_copy_region(struct boot_loader_state *state,
+                     const struct flash_area *fap_src,
+                     const struct flash_area *fap_dst,
+                     uint32_t off_src, uint32_t off_dst, uint32_t sz);
+int boot_erase_region(const struct flash_area *fap, uint32_t off, uint32_t sz);
+int boot_status_init(const struct boot_loader_state *state,
+                     const struct flash_area *fap,
+                     const struct boot_status *bs);
+void boot_perform_swap(struct boot_loader_state *state,
+                       struct boot_status *bs,
+                       uint32_t copy_size);
+int boot_status_source(struct boot_loader_state *state);
+int boot_read_status_bytes(const struct flash_area *fap,
```

Comment:
What are 'status bytes' and where are they read from? Which parameter (or return value?) contains the status bytes after I've called this?